src/components/Icons/Icons.cy.tsx

```diff
@@ -1,65 +1,112 @@
 import React from "react";
 
 import CongestionChargeAppliedIcon from "@/components/Icons/CongestionChargeAppliesIcon";
-import FeetIcon from "@/components/Icons/FeetIcon";
-import FlagIcon from "@/components/Icons/FlagIcon";
+import FeetIcon, { FeetIconProps } from "@/components/Icons/FeetIcon";
 import PhoneIcon from "@/components/Icons/PhoneIcon";
-import SpeechBubbleIcon from "@/components/Icons/SpeechBubbleIcon";
+import SpeechBubbleIcon, { SpeechBubbleProps } from "@/components/Icons/SpeechBubbleIcon";
 import TruckIcon from "@/components/Icons/TruckIcon";
+import StyleManager from "@/app/themes";
+import FlagIcon from "@/components/Icons/FlagIcon";
+
+const StyledCongestionChargeAppliedIcon: React.FC = () => {
+    return (
+        <StyleManager>
+            <CongestionChargeAppliedIcon />
+        </StyleManager>
+    );
+};
+const StyledFeetIcon: React.FC<FeetIconProps> = (props) => {
+    return (
+        <StyleManager>
+            <FeetIcon {...props} />
+        </StyleManager>
+    );
+};
+
+const StyledFlagIcon: React.FC = () => {
+    return (
+        <StyleManager>
+            <FlagIcon />
+        </StyleManager>
+    );
+};
+const StyledPhoneIcon: React.FC = () => {
+    return (
+        <StyleManager>
+            <PhoneIcon />
+        </StyleManager>
+    );
+};
+
+const StyledSpeechBubbleIcon: React.FC<SpeechBubbleProps> = (props) => {
+    return (
+        <StyleManager>
+            <SpeechBubbleIcon {...props} />
+        </StyleManager>
+    );
+};
+
+const StyledTruckIcon: React.FC = () => {
```

Comment:
Add the explicit type parameter to React.FC everywhere in this file

Reply:
Done